internal/search/query/types.go

```diff
@@ -129,6 +129,19 @@ func (b Basic) GetCount() string {
 	return countStr
 }
 
+// GetTimeout returns the string value of the "timeout:" field. Returns empty string if none.
+func (b Basic) GetTimeout() *time.Duration {
+	var timeout *time.Duration
+	VisitField(ToNodes(b.Parameters), "timeout", func(value string, _ bool, _ Annotation) {
```

Comment:
Just to make "find references" a bit easier. 
```suggestion
	VisitField(ToNodes(b.Parameters), FieldTimeout, func(value string, _ bool, _ Annotation) {
```